[2024-05-01T16:03:21.922+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-01T16:03:21.952+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: wikipedia_flow.transform_wikipedia_data manual__2024-05-01T16:03:13.237211+00:00 [queued]>
[2024-05-01T16:03:21.960+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: wikipedia_flow.transform_wikipedia_data manual__2024-05-01T16:03:13.237211+00:00 [queued]>
[2024-05-01T16:03:21.961+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-01T16:03:21.971+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): transform_wikipedia_data> on 2024-05-01 16:03:13.237211+00:00
[2024-05-01T16:03:21.975+0000] {standard_task_runner.py:63} INFO - Started process 3559 to run task
[2024-05-01T16:03:21.978+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'wikipedia_flow', 'transform_wikipedia_data', 'manual__2024-05-01T16:03:13.237211+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/wikipedia_flow.py', '--cfg-path', '/tmp/tmpg4h_c_wh']
[2024-05-01T16:03:21.979+0000] {standard_task_runner.py:91} INFO - Job 20: Subtask transform_wikipedia_data
[2024-05-01T16:03:22.025+0000] {task_command.py:426} INFO - Running <TaskInstance: wikipedia_flow.transform_wikipedia_data manual__2024-05-01T16:03:13.237211+00:00 [running]> on host ab8381b78bd9
[2024-05-01T16:03:22.211+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Minh Tran' AIRFLOW_CTX_DAG_ID='wikipedia_flow' AIRFLOW_CTX_TASK_ID='transform_wikipedia_data' AIRFLOW_CTX_EXECUTION_DATE='2024-05-01T16:03:13.237211+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-01T16:03:13.237211+00:00'
[2024-05-01T16:03:22.212+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-01T16:20:27.307+0000] {logging_mixin.py:188} WARNING - /opt/***/pipelines/wikipedia_pipeline.py:110 SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2024-05-01T16:20:27.342+0000] {python.py:237} INFO - Done. Returned value was: OK
[2024-05-01T16:20:27.342+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-01T16:20:27.367+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=wikipedia_flow, task_id=transform_wikipedia_data, execution_date=20240501T160313, start_date=20240501T160321, end_date=20240501T162027
[2024-05-01T16:20:27.390+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-01T16:20:27.493+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-01T16:20:27.494+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
